wandb/apis/public.py

```diff
@@ -2637,7 +2637,11 @@ def get(
             query = cls.LEGACY_QUERY
             response = client.execute(query, variable_values=variables)
 
-        if not response or not response.get("project", {}).get("sweep"):
```

Comment:
this line is actually bugged! when res = {"project": None} we get a none type err